### PR TITLE
Update SASS Style.

### PR DESCRIPTION
**Pull request by leostudiooo**
1. Update `font-family`
2. Set a **variable** *basic* `z-index` named `$least-zindex`

### CODE REVIEW

File: src/sass/player.sass [hunk 1]
```diff
@@ -6,14 +6,16 @@
     to
         transform: rotate(360deg)
 
+$least-zindex: 100
```
Comment (LuoTR) [inline src/sass/player.sass:9]: Why define variable here instead of `_variables.sass`?

**Comment by leostudiooo** [comment]
well at first i defined it in _variable.sass but it throws an error then. so i have to define it here instead.

**Comment by LuoTR** [comment]
> well at first i defined it in _variable.sass but it throws an error then. so i have to define it here instead.

What error was it?

**Comment by leostudiooo** [comment]
and for another reason,  it's easier to change the base figure here. (though it's just nonsense):-D

**Comment by leostudiooo** [comment]
let me check.


-------- Original message --------
From: LRain ***@***.***>
Date: Tue, May 4, 2021, 01:01
To: M4TEC/PenguinPlayer ***@***.***>
Cc: Leo Lingfeng Li ***@***.***>, Author ***@***.***>
Subject: Re: [M4TEC/PenguinPlayer] Update SASS Style. (#18)

well at first i defined it in _variable.sass but it throws an error then. so i have to define it here instead.

What error was it?

―

**Comment by leostudiooo** [comment]
image
```
ERROR in ./src/sass/player.sass (./node_modules/_css-loader@5.2.0@css-loader/dist/cjs.js!./node_modules/_postcss-loader@5.2.0@postcss-loader/dist/cjs.js!./node_modules/_sass-loader@11.0.1@sass-loader/dist/cjs.js!./src/sass/player.sass)
Module build failed (from ./node_modules/_sass-loader@11.0.1@sass-loader/dist/cjs.js):
SassError: Undefined variable.
   ╷
17 │         z-index: $least-zindex + 2
   │                  ^^^^^^^^^^^^^
   ╵
  src\sass\player.sass 17:18  root stylesheet
SassError: SassError: Undefined variable.
   ╷
17 │         z-index: $least-zindex + 2
   │                  ^^^^^^^^^^^^^
   ╵
  src\sass\player.sass 17:18  root stylesheet
    at C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass-loader@11.0.1@sass-loader\dist\index.js:54:16
    at Function.call$2 (C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass@1.32.8@sass\sass.dart.js:91729:16)
    at _render_closure1.call$2 (C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass@1.32.8@sass\sass.dart.js:80373:12)
    at _RootZone.runBinary$3$3 (C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass@1.32.8@sass\sass.dart.js:27269:18)
    at _FutureListener.handleError$1 (C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass@1.32.8@sass\sass.dart.js:25797:19)
    at _Future__propagateToListeners_handleError.call$0 (C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass@1.32.8@sass\sass.dart.js:26094:49)
    at Object._Future__propagateToListeners (C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass@1.32.8@sass\sass.dart.js:4543:77)
    at _Future._completeError$2 (C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass@1.32.8@sass\sass.dart.js:25927:9)
    at _AsyncAwaitCompleter.completeError$2 (C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass@1.32.8@sass\sass.dart.js:25270:12)
    at Object._asyncRethrow (C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass@1.32.8@sass\sass.dart.js:4292:17)
 @ ./src/sass/player.sass 2:12-241 9:17-24 13:15-29
 @ ./src/typescript/player.ts 12:0-29

webpack 5.30.0 compiled with 1 error in 4539 ms
npm ERR! code 1
npm ERR! path C:\Users\Leo\Documents\GitHub\penguinplayer
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c webpack --mode=production --env production
```

**Comment by leostudiooo** [comment]
but here I actually set this variable to 100.

**Comment by LuoTR** [comment]
> image
> 
> ```
> ERROR in ./src/sass/player.sass (./node_modules/_css-loader@5.2.0@css-loader/dist/cjs.js!./node_modules/_postcss-loader@5.2.0@postcss-loader/dist/cjs.js!./node_modules/_sass-loader@11.0.1@sass-loader/dist/cjs.js!./src/sass/player.sass)
> Module build failed (from ./node_modules/_sass-loader@11.0.1@sass-loader/dist/cjs.js):
> SassError: Undefined variable.
>    ╷
> 17 │         z-index: $least-zindex + 2
>    │                  ^^^^^^^^^^^^^
>    ╵
>   src\sass\player.sass 17:18  root stylesheet
> SassError: SassError: Undefined variable.
>    ╷
> 17 │         z-index: $least-zindex + 2
>    │                  ^^^^^^^^^^^^^
>    ╵
>   src\sass\player.sass 17:18  root stylesheet
>     at C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass-loader@11.0.1@sass-loader\dist\index.js:54:16
>     at Function.call$2 (C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass@1.32.8@sass\sass.dart.js:91729:16)
>     at _render_closure1.call$2 (C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass@1.32.8@sass\sass.dart.js:80373:12)
>     at _RootZone.runBinary$3$3 (C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass@1.32.8@sass\sass.dart.js:27269:18)
>     at _FutureListener.handleError$1 (C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass@1.32.8@sass\sass.dart.js:25797:19)
>     at _Future__propagateToListeners_handleError.call$0 (C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass@1.32.8@sass\sass.dart.js:26094:49)
>     at Object._Future__propagateToListeners (C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass@1.32.8@sass\sass.dart.js:4543:77)
>     at _Future._completeError$2 (C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass@1.32.8@sass\sass.dart.js:25927:9)
>     at _AsyncAwaitCompleter.completeError$2 (C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass@1.32.8@sass\sass.dart.js:25270:12)
>     at Object._asyncRethrow (C:\Users\Leo\Documents\GitHub\penguinplayer\node_modules\_sass@1.32.8@sass\sass.dart.js:4292:17)
>  @ ./src/sass/player.sass 2:12-241 9:17-24 13:15-29
>  @ ./src/typescript/player.ts 12:0-29
> 
> webpack 5.30.0 compiled with 1 error in 4539 ms
> npm ERR! code 1
> npm ERR! path C:\Users\Leo\Documents\GitHub\penguinplayer
> npm ERR! command failed
> npm ERR! command C:\Windows\system32\cmd.exe /d /s /c webpack --mode=production --env production
> ```

Well, it's your problem, check another variable usage in this file.

**Comment by leostudiooo** [comment]
Shit, I forgot to use the padding `variables.` before `$least-zindex`. Will close this PR and open another.

**Pull request closed without merging**